spv/go.mod

```diff
@@ -0,0 +1 @@
+module github.com/decred/dcrwallet/spv/v3
```

Comment:
This could/would cause confusion/conflicts. Why not `module github.com/raedahgroup/dcrlibwallet/spv`?